content/blog/2020/hello-world.md

```diff
@@ -3,7 +3,8 @@ title: "Hello World!"
 description: "Getting started using the BDK library in a very simple Rust project"
 authors: 
     - Alekos Filini
-date: "2020-12-18"
+    - Claudio Ciardelli
```

Comment:
For doing an update like this I don't think you should change the article date or add your name here, this should stay as the original author(s) and published date.  

But you should add a line at the end of the file to indicate that the article was updated for `bdk v0.11`, something like:

_Revised 2021-09 to support API changes in BDK version `v0.11.0`._

Reply:
Fixed